plugins/dagsvisualizer/visualizer.go

```diff
@@ -159,9 +160,20 @@ func registerBranchEvents() {
 		})
 	})
 
+	branchWeightChangedClosure := events.NewClosure(func(e *tangle.BranchWeightChangedEvent) {
+		visualizerWorkerPool.TrySubmit(&wsMessage{
+			Type: MsgTypeBranchWeightChanged,
+			Data: &branchWeightChanged{
+				ID:     e.BranchID.Base58(),
+				Weight: e.Weight,
```

Comment:
When weight changes, GoF is changed, too.
Let's grab the new GoF and include it here.